### ISSUE TITLE
Feature Request：a module that controls the learning rate of its containing modules

### ISSUE THREAD

**Issue by Naruto-Sasuke** [torch/nn#1235]
By this type of module, we can set different learning rates of different parts of the net.
It is quite useful when we want to finetune the former part of the net and change the latter part dramatically.
